README.md

```diff
@@ -10,8 +10,64 @@ The website uses the LAA Legal Adviser API. The code is hosted at https://github
 
 The REST API is https://prod.laalaa.dsd.io/.
 
+## Dependencies
+- [docker](https://www.docker.com/)
+
+## Installation for development
+
+### Tips for developing with docker containers
+
+Code editing - You can edit the code on your local disk, with a local editor, as normal. (You don't have to edit the 
+files inside the Docker container, because your local directory is mounted into container.) When you save a file, it 
+becomes present in the container immediately, and the server restarts.
+
+Browsing the app - Point your local browser at http://localhost:8013/
+
+Log output - watch the output generated by the running app using: `docker logs fala -f` or `docker attach fala`
+
+Please ensure your internet network such as your VPN does not interfere with your build.
+
+To run container:
+
+`./run_local.sh`
+
+This shell script contains commands that build and runs the fala app.
+
+If you're not using `Docker Desktop`, you may want to have two terminals open for the following:
+
+1. To `exec` into the fala docker container.
+
+    `docker exec -it fala bash`
+
+    From the shell inside the container you can run some tests e.g.
+
+    `python manage.py test`
+
+2. Logs and debugging.
+
+    You can see the logging output by attaching to the container:
+
+    `docker attach fala`
+
+    If you added pdb breakpoints, this is where you can interact with this debugger.
+
+Alternatively, some editors have functionality to hook into running containers, such as VS Code's 'Dev Containers' 
+extension. Docker Desktop offers a UI version of looking through logs, exec and many other interactions. 
+
+### Debugging
+Depending on how you're running the project, via `venv` or Docker you can perform the following debugging:
+
+Add import pdb; pdb.set_trace() as a 'breakpoint' line in the code, where you want to trigger the debugger.
```

Comment:
The docs say either works https://docs.python.org/3/library/pdb.html, but in Python3 now, it's generally preferred to use `breakpoint()`, instead of `import pdb; pdb.set_trace()`

Reply:
Also, for debugging to work in docker, you need to add this:
```
diff --git a/docker-compose.yml b/docker-compose.yml
index feaaa39..9a0b816 100644
--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -17,3 +17,4 @@ services:
       DJANGO_SETTINGS_MODULE: "fala.settings.local"
     ports:
       - "8013:8000"
+    stdin_open: true
```